extra/3-magic-8-ball.js

```diff
@@ -101,7 +114,9 @@ test("magic 8 ball returns different values each time", () => {
     );
   }
 
-  let seenPositivities = new Set(Array.from(seenAnswers.values()).map(checkAnswer));
+  let seenPositivities = new Set(
+    Array.from(seenAnswers.values()).map(checkAnswer)
+  );
```

Comment:
There isn't an elegant way to do this - one option is to put all the possible answers into 1 array in order, get one at random, and the use the index to decide whether it is very positive, positive etc. Once you learn about objects in a future lesson, you could find a better way of doing this.

Reply:
this not my code, this is code from jest. my solution start from line 67 to 76